Hazel/src/Hazel/Core/PlatformDetection.h

```diff
@@ -33,7 +33,6 @@
 	#error "Android is not supported!"
 #elif defined(__linux__)
 	#define HZ_PLATFORM_LINUX
```

Comment:
This is a Linux branch which should be in working order. The `#error` preprocessor directive raised by `HZ_PLATFORM_LINUX` shouldn't be in a branch that is effectively a prototype or recipe for Cherno to use in future videos. This branch should be a working state that is ready to clone.

Reply:
I do want to merge this PR, since it has nothing 'new' for Linux compatibility other than some minor tweaks to the code without actually changing any behaviour... So yes, it 'should' be in working order, but when I merge this, Hazel will still not officially support it. Those that follow along on linux can easily delete this one line and it should work for them, rather than requiring to make many changes to the code base...

Reply:
@LovelySanta it works, what's wrong with it?

Reply:
That it officially isn't supported, so if someone downloads it from github, it's supposed to say it's not compatible, hence why the error must stay. However, you could add a comment that when you remove this error line, it should run unofficially